### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by ahemphill**

### CODE REVIEW

File: tools/scoring/README.md [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # Scoring
 
-The scoring tool evaluates _proposed_ configuration files against "known good" _solution_ files. It uses an F-score algorithm to provide floating-point representations of how closely the files compare along various _dimensions_.
+The scoring tool evaluates _proposed_ configuration files against "known good" _solution_ files. It uses an F-score algorithm to provide floating-point representations of how closely the files compare along various [_dimensions_](#dimensions). Broadly, its process is thus:
+
+1. Filter out irrelevant entities (e.g. those which have a noncanonical type)
+2. Isolate the relevant attribute(s) of each entity and reduce them to sets (e.g. [connections](https://github.com/google/digitalbuildings/blob/master/ontology/docs/ontology_config.md#connections))
+3. Count the `intersection` and `difference` between the respective _proposed_ and _solution_ sets
+4. Input these figures to the aforementioned scoring algorithm to produce a singular numeric score
```
Comment (charbull) [inline tools/scoring/README.md:8]: this is cool ! thank you !
can you please mention how reporting entities vs virtual entities are calculated?

**Review by charbull — CHANGES_REQUESTED** [review]
just one comment  :) 
Looks pretty good

File: tools/scoring/README.md (hunk 1 above)
Comment (ahemphill) [reply tools/scoring/README.md]: https://github.com/google/digitalbuildings/pull/549/commits/af74fa8af4c2ca43b58c7a75686cc04983d4961c

**Review by charbull — CHANGES_REQUESTED** [review]
this is duplicated?

**Comment by ahemphill** [comment]
No, the other PR is just branched from this.

File: tools/scoring/README.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
 # Scoring
 
-The scoring tool evaluates _proposed_ configuration files against "known good" _solution_ files. It uses an F-score algorithm to provide floating-point representations of how closely the files compare along various _dimensions_.
+The scoring tool evaluates _proposed_ configuration files against "known good" _solution_ files. It uses an F-score algorithm to provide floating-point representations of how closely the files compare along various [_dimensions_](#dimensions). Broadly, its process is thus:
```
[the remark this answers is not in the document]
Comment (ahemphill) [reply tools/scoring/README.md]: https://github.com/google/digitalbuildings/pull/549/commits/b97384fb499100da6f85080782d2fcd1bf89d237

**Pull request merged**